content/reference/plugins/jenkins/index.md

```diff
@@ -25,13 +25,27 @@ You need to be connected as an administrator of your Jenkins application to inst
 
 {{< img src="installation.png" alt="Installation" >}}
 
+## API Token and Jenkins credentials
+
+This plugin requires an [API token]({{< ref "../../admin/api_tokens" >}}) to allow Jenkins to authenticate with Gatling Enterprise. The API token needs the Start permission.
+
+We recommend storing the API token using [Jenkins credentials](https://www.jenkins.io/doc/book/using/using-credentials/). Go to **Manage Jenkins**, then **Manage credentials**. You will see your existing credentials, as well as the credentials stores and domains configured on your Jenkins instance.
+
+{{< img src="manage-credentials.png" alt="Manage credentials" >}}
+
+To add new credentials, click on the name of the domain you want to use, then on the **Add Credentials** button. Choose the type **Secret text** and the scope you want to restrict the credentials to, copy and paste your API token to the **Secret** field, and enter an ID (and optionally a description). Click on **Create**.
+
+{{< img src="new-credentials.png" alt="New credentials" >}}
+
+{{< alert warning >}}
+If you don't use Jenkins credentials, you can still set an API token directly in the plugin global configuration, but this is not as secure as using the Jenkins credentials and is not recommended.
+{{< /alert >}}
+
 ## Configuration
 
 The plugin needs some global configuration. Go to **Manage Jenkins**, **Configure System**, then **Global Gatling Enterprise Plugin Configuration**.
 
-The [API token]({{< ref "../../admin/api_tokens" >}}) will allow Jenkins to authenticate to Gatling Enterprise. The API token needs the Start permission.
-
-You can configure the API Token globally in this page, or per CI project if each project has an API Token scoped on a specific team. We recommend storing the API Token in a secret text credential, but you can also copy the content of the API Token in the second field.
+Choose the Jenkins credentials where [you stored your API token]({{< ref "#api-token-and-jenkins-credentials" >}}). Alternatively, you can just copy the content of the API Token to the second field, but this is not recommended.
```

Comment:
Should we mention options we don't recommend (and that objectively shouldn't be used)?
Shouldn't we stop advertising them and start deprecating them for removal in the code?

Reply:
That's a good question. Since the field shows up in the UI, it's a bit weird to ignore it completely I think? Maybe keep the warning notice in the "API Token and Jenkins credentials" just to give some context, but remove this mention here, which actually explains how to do it (since you shouldn't do it)? 🤔

Note: the field title in the UI includes the word DEPRECATED in all caps, so I'd say there is already some warning that this may be eventually removed.

Reply:
I made that change; I'm going to merge this PR, and do the same for the self-hosted doc.